engine-src/Game/LambdaHack/Common/Misc.hs

```diff
@@ -102,9 +102,11 @@ appDataDir = do
   let name = takeWhile Char.isAlphaNum progName
   getAppUserDataDirectory name
 
+-- | multiply by a million
```

Comment:
For whatever arbitrary reason, I write full sentences, with a dot, except for comments that are on the same line as code (or on the next line if not enough space) and refer to this line in particular.